### DISCUSSION TITLE
Hide files from showing up in the GitHub online viewer

### DISCUSSION THREAD

**Discussion by hughbreslin** [community/community#39342] [Repositories]
### Select Topic Area

Product Feedback

### Body

Does anyone know of a way (either through repository configuration or an org level profile) to hide certain directories from being displayed in the GitHub online repository browser? We host docs in our GitHub org and have a images dotted throughout and would like to hide the images folder as its redundant when you're the consumer of the docs and just adds folders in the viewer. 

Our structure is
```
Document_name/
├─ images/
│  ├─ image.png
├─ Document.md
```

It would be nice to hide the "Images" directory in the repo browser online but still have it present when the repository is cloned.

**Comment by airtower-luna** [comment]
Simple answer: No, that's not possible. If you want to have a website (rather than technical) view, you might want to consider using Pages to publish an HTML rendering of the Markdown.

**Reply by hughbreslin** [reply]
Yeah I was thinking that was the answer but worth a try! It’s handy for us to have the markdown in the same org as the sources as it’s just easy to see the docs there - we could go the pages route as you mentioned

**Reply by airtower-luna** [reply]
You can keep the Markdown files right there, you don't need to move them. I just mean you can publish the rendered version for easy reading. :slightly_smiling_face:
